packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -782,7 +782,10 @@ function mountLazyComponent(
   workInProgress.type = Component;
   const resolvedTag = (workInProgress.tag = resolveLazyComponentTag(Component));
   startWorkTimer(workInProgress);
-  const resolvedProps = resolveDefaultProps(Component, props);
+  const resolvedProps = (workInProgress.pendingProps = resolveDefaultProps(
+    Component,
+    props,
```

Comment:
The type and tag are overwritten, so my first thought was that the same should be done for `pendingProps`